### DISCUSSION TITLE
Friends can't join me with external IP

### DISCUSSION THREAD

**Discussion by TonZaga** [wolveix/satisfactory-server#106] [General]
I'm able to setup my server in Docker and forwarded ports (15777, 15000, 7777 udp) but when they use my external IP, they are getting "server name pending", "offline", and "notauthenticated".  Is this still a router issue or something I can change in the config?

**Comment by wolveix** [comment]
Hey. Please open this as an issue in the future, as discussions are not intended for this purpose.

Regarding your query, it's most likely that this is a networking issue. Make sure that you've forwarded all of the ports to your host machine via your router. You can check that your ports are publicly visible by using a tool like https://www.yougetsignal.com/tools/open-ports/
